### PR TITLE
enhancement: add python packages

### PR DESCRIPTION
**Pull request by aakropotkin**
Also adds a slim wrapper around `flox publish` to avoid repetitive strain injuries.

### CODE REVIEW

**Review by mkenigs — COMMENTED** [review]
Since the re-exports are coming from non-capacitated flakes, should you be doing `inputs.$pkg` instead of `capacitated.$pkg`? (I dunno, @ysndr would)

Personal preference, but I would inline all these packages in flake.nix rather than creating a bunch of 4 line files.

**Comment by aakropotkin** [comment]
@mkenigs idk @limeytexan wrote those. I just made a fix to the inputs/follows and added the wrapper script

**Comment by ysndr** [comment]
> Since the re-exports are coming from non-capacitated flakes, should you be doing inputs.$pkg instead of capacitated.$pkg? (I dunno, @ysndr would)

I the reexport all seem to be ordinary capacitated flakes -- except `flox-examples/poetry-qiskit` which has a rather creative way to define packages.

File: flake.nix [hunk 1]
```diff
@@ -1,23 +1,100 @@
 {
-  
+  description = "Floxpkgs/Project Template";
```
Comment (ysndr) [inline flake.nix:2]:
```suggestion
  description = "Package set of examples packaged for the use with flox";
```

Comment (ysndr) [reply flake.nix:2]: I think this repo deserves a correct description, whether we do this in this PR or not and use my suggestion or something else I'll leave to you.

File: flake.nix [hunk 2]
```diff
@@ -1,23 +1,100 @@
...
+          #
+          # Note: composing package sets may require rebuilding some packages
+          #       as some dependency versions may differ
+          #       from one package set to another.
+          #       composing such package sets ensures a single coherent set.
+          projects = {
+            /*
+            inherit (context.capacitated) my-project;
+            */
+          };
```
Comment (ysndr) [inline flake.nix:84]: not blocking: I don't know if we're importing entire package sets here with `py-roberto` and `poetry-qiskit` but if we do we can just mix them in here.

File: pub.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  exit 1;
+fi
+
+: "${BRANCH:=$( $GIT -C "$SDIR" branch --show-current; )}";
+
+# Check for dirty tree
+if ! $GIT -C "$SDIR" diff "origin/$BRANCH" --quiet 2>/dev/null; then
+  echo "pub.sh: Tree is dirty or unpushed. Commit/push, and try again." >&2;
+  exit 1;
+fi
```
Comment (ysndr) [inline pub.sh:33]: `flox publish` should check this already..

File: pub.sh [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+# Utilities
+: "${NIX:=nix}";
+: "${FLOX:=flox}";
+: "${GIT:=git}";
+: "${REALPATH:=$NIX shell nixpkgs#coreutils -c realpath}";
+
+# Default settings
+: "${OWNER:=flox-examples}";
+: "${REPO:=floxpkgs}";
+: "${STABILITY:=stable}";
```
Comment (ysndr) [inline pub.sh:17]: Does `$FLOX_STABILIT` not work for you?

Comment (aakropotkin) [reply pub.sh:17]: I didn't know about that variable.

**Comment by aakropotkin** [comment]
I just dropped the helper script since it seemed to be redundant.

File: flake.nix (hunk 2 above)
Comment (aakropotkin) [reply flake.nix:84]: I don't know how

**Review by ysndr — APPROVED** [review]

**Pull request merged**